hadoop-common-project/hadoop-common/src/main/java/org/apache/hadoop/conf/Configuration.java

```diff
@@ -18,6 +18,7 @@
 
 package org.apache.hadoop.conf;
 
+
```

Comment:
Avoid

Reply:
This change is to run spotbugs check in the precommit Jenkins job. Yes, I'll commit after reverting this.